### ISSUE TITLE
Issues with onlineconf "Create chat" and "Chat list" tab

### ISSUE THREAD

**Issue by mdempsky** [urweb/upo#19]
A few issues I noticed playing with the chat functionality:

1. In the "Create chat" tab, after clicking "Add", there's no success indication.

2. It's not obvious what format the start time needs to be in. It would probably be best to actually use a date/time picker widget in this case? (Unfortunately, `<input type="datetime-local">` still doesn't seem well supported yet: https://caniuse.com/#search=datetime-local)

3. Navigating to the "Chat list" tab does not list the newly created chat until after a reload.

**Comment by achlipala** [comment]
The first two of those points I've just fixed by updating this example with more of what I was already using in larger examples (outside this repo).

The last point is still present, w.r.t. the "Chat list" and "Scheduling chats" tabs.  I wonder if it would be a useful getting-started puzzle for you to figure out how to fix the library so that the proper notifications go between those tabs.  I believe "Chat list" should already be behaving properly, while "Scheduling chats" needs to be updated to send notifications.

The latter is using library module `SetTimes`, which I didn't equip with change notification yet.  The more-complex module `AssignTimes` *does* do such notification, and I imagine it's possible to crib ideas from the latter to bring to the former.

So... mission accepted? :)

**Comment by mdempsky** [comment]
Yeah, I'm interested in digging into these issues. I just figured I'd start by poking around and taking note of what I observe as a first-time user, before I jumped too much into the implementation details and became "tainted" by that knowledge.
